data-jdbc/src/test/groovy/io/micronaut/data/jdbc/h2/H2ValidationSpec.groovy

```diff
@@ -56,6 +57,8 @@ class H2ValidationSpec extends Specification {
         foodRepository.searchById(food.fid)
     }
 
+    // TODO: Add back later
+    @Ignore("Ignored until validation is fixed")
```

Comment:
Are we good to go with this until we fix this so test resources don't have issue?

Reply:
Yeah sure, I will fix it later